core/src/main/scala/org/ensime/indexer/orientdb/SimpleOrient.scala

```diff
@@ -32,9 +34,77 @@ package api {
   case class VertexT[T](underlying: Vertex)
 
   trait EdgeT[+Out, +In]
+
+  sealed trait IndexT {
+    def orientKey: String
+  }
+
+  object IndexT {
```

Comment:
same comment as above about this `sealed trait` with implementations in the companion pattern, it's a bit unusual.